plio/settings.py

```diff
@@ -329,15 +329,27 @@
     import sentry_sdk
     from sentry_sdk.integrations.django import DjangoIntegration
 
+    def strip_sensitive_data(event, hint):
+        """Strips user email and username from the event/error details. Only data remains is event['user']['id']."""
+        try:
+            event["user"].pop("email")
+            event["user"].pop("username")
```

Comment:
there is no username field for us right? 
We should also add `mobile` here right?

Reply:
Yes. it's not there but it's set by default with `django.contrib.auth`. Just as a precaution, I've deliberately popped that element. If it's not present, the try/catch block will take care.